src/common/strvararg.cpp

```diff
@@ -703,52 +703,55 @@ wxFormatString::ArgumentType DoGetArgumentType(const CharType *format,
         return wxFormatString::Arg_Unused;
     }
 
-    wxCHECK_MSG( parser.pspec[n-1] != NULL, wxFormatString::Arg_Unknown,
-                 "requested argument not found - invalid format string?" );
-
-    switch ( parser.pspec[n-1]->m_type )
+    if (0<n)
```

Comment:
I could be wrong, but I don't think `n` could be 0 here, it's the index of the argument and there must be the format string before. So IMO this change is really unneeded and it would be nice to find some way to pacify Coverity without introducing it.

Of course, disregard this if you can actually reproduce the problem in this code (how?).